src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Process_StreamTests.cs

```diff
@@ -44,7 +44,10 @@ public void Process_AsyncErrorStream()
             p.ErrorDataReceived += (s, e) => { sb.Append(e.Data); };
             p.Start();
             p.BeginErrorReadLine();
-            p.WaitForExit(WaitInMS);
+
+            if (p.WaitForExit(WaitInMS))
```

Comment:
We could do this separately, but I wonder if anywhere we do ```p.WaitForExit(WaitInMS)``` we should actually do ```Assert.True(p.WaitForExit(WaitInMS))```.   Presumably we believe WaitInMS is set such that there's no situation in which these calls would ever return false, or else this and other logic in these tests is buggy.  Then this would just be:
```C#
Assert.True(p.WaitForExit(WaitInMS));
p.WaitForExit();
```